src/nix/develop.cc

```diff
@@ -507,7 +507,7 @@ struct CmdDevelop : Common, MixEnvironment
                 state,
                 installable->nixpkgsFlakeRef(),
                 "bashInteractive",
-                DefaultOutputs(),
+                OutputNames{"out"},
```

Comment:
It will also break if `bashInteractive` is ever renamed, and there’s a big fat warning comment about that in Nixpkgs, so I assumed that would be okay? The warning should probably be extended to mention the output name as well, of course.